scripts/globals/abilities/call_beast.lua

```diff
@@ -25,4 +25,5 @@ end
 function onUseAbility(player, target, ability)
     tpz.pet.spawnPet(player, player:getWeaponSubSkillType(tpz.slot.AMMO))
     player:removeAmmo()
+    player:addRecast(tpz.recast.ABILITY, 102, 1)
```

Comment:
Is this 102 the entry in `abilities_charges`? I'd say leave a small comment explaining where to find it, it isn't a well known table